autoemulate/experimental/data/utils.py

```diff
@@ -110,6 +110,9 @@ def _convert_to_tensors(
                 return x.to(dtype)
             msg = "Number of tensors returned must be greater than zero."
             raise ValueError(msg)
+        # TODO: this error message will never be raised because of
+        # of the value error in _convert_to_dataset
+        # If I remove it, Ruff complains that a return statement is missing
         raise ValueError(
```

Comment:
This error message will never be raised because of the value error in `_convert_to_dataset`, however, if I remove it, Ruff complains that a return statement is missing